package.json

```diff
@@ -4,7 +4,8 @@
   "description": "This is the indexer solution for the xion chain",
   "main": "dist/index.js",
   "scripts": {
-    "build": "subql build",
+    "build": "rm -rf dist && subql build",
+    "generate:proto:types": "protoc --plugin=./node_modules/.bin/protoc-gen-ts_proto --ts_proto_out=src/types/ --ts_proto_opt=useDate=false --proto_path proto $1",
```

Comment:
What does `$1` mean here?

Reply:
Should be explicit. It's the first argument passed to the command. It should be the proto file to compile

Reply:
Can you add that to the readme?